example_cluster/setup_fake_config.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+import shutil
+import socket
+import subprocess
+import sys
+from distutils.dir_util import copy_tree
+
+
+def find_open_ports():
```

Comment:
`paasta_tools.cli.utils.pick_random_port`

Reply:
ephemeral_port_reserve  is not found even though it's in requirements.txt... It works if I run the script with .tox/py36-linux/bin/activate && python -m ...

Reply:
We can pair together to fix this if you want, but this represents a stale venv. You can `rm -rf .paasta` and make `make` recreate it to get an updated venv.